pallets/loans/src/mock.rs

```diff
@@ -129,17 +132,37 @@ impl pallet_balances::Config for Runtime {
     type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
 }
 
+lazy_static! {
+    pub static ref PRICE_ORACLE: Mutex<HashMap<CurrencyId, Option<PriceDetail>>> = {
+        Mutex::new(
+            vec![DOT, KSM, BTC, USDT, XDOT]
+                .iter()
+                .map(|&x| (x, Some((1, 1))))
+                .collect(),
+        )
+    };
+}
+
+impl PRICE_ORACLE {
```

Comment:
How about `MockPriceFeeder`?

Reply:
Good idea, I will do that.

Reply:
I will rename it to `MOCK_PRICE_FEEDER` because the static variable should have an upper case name.